src/lib/routes/util.ts

```diff
@@ -30,18 +30,24 @@ export const handleErrors: (
     const finalError =
         error instanceof UnleashError ? error : fromLegacyError(error);
 
+    const format = (thing: object) => JSON.stringify(thing, null, 2);
+
     if (!(error instanceof UnleashError)) {
         logger.debug(
-            `I encountered an error that wasn't an instance of the \`UnleashError\` type. The original error and what it was mapped to are:`,
-            error,
-            finalError,
+            `I encountered an error that wasn't an instance of the \`UnleashError\` type. The original error was: ${format(
+                error,
+            )}. It was mapped to ${format(finalError.toJSON())}`,
         );
     }
 
-    logger.warn(finalError.id, finalError.message);
+    logger.warn(
+        `Error message: "${finalError.message}" Error ID: "${
+            finalError.id
+        }". Full, serialized error: ${format(finalError.toJSON())}`,
+    );
 
     if (['InternalError', 'UnknownError'].includes(finalError.name)) {
-        logger.error('Server failed executing request', error);
+        logger.error(`Server failed executing request: ${format(error)}`);
```

Comment:
Okay! Maybe that's more useful, then. I'm happy to revert that line if that's better. The format function is defined a few lines before and just formats it with json.stringify (to avoid logging [Object.object]